docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -97,164 +83,204 @@ can be seen in the Information tab, as shown in the image below. For more inform
-  So for the test setup the **Timeout is set to 1.5s**. Depending on how frequently your flight controller sends throttle commands, you may want to decrease this when using the motors on a drone.
+**However, because this test is meant to simulate commands from a flight controller, it is affected by the** :ref:`manual_advanced_arming` **feature. That means that setting this parameter 
+will not cause the module to spin until the module has armed.** By default Vertiq speed modules require 10 consecutive throttle commands between 0% and 7.5% to arm. So to arm your module using the
+*ESC Input* parameter, set the *ESC Input* to 0.05 to send a 5% throttle command as shown below, and click the set arrow 10 times. On the 10th time you click the set arrow, the module should play 
+its 2 note arming song, and begin spinning. 
 
-Testing the Motor
-*****************
-.. warning:: Double check that the motor is secured and there is no propeller attached before performing any testing.
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/control_center_esc_input.png
+    :align: center
+
+    ESC Input Parameter For 5% Throttle Command
 
-To test that motor is configured to spin and take throtle commands from the flight controller, open the Testing tab. The ESC Input parameter simulates the motor receiving a 
-throttle command, so it is the best way to test if the motor is ready to connect with the flight controller. **Make sure the motor is secure before attempting to spin it, as
-it will "jump" if unsecured**. Set the ESC Input to 0.1. The motor should start spinning. Next, set the ESC Input to 0.2. Finally, The motor should start spinning more quickly. Check
-that the motor is spinning counter-clockwise. Finally, set the ESC Input to 0.6. The motor should speed up and continue spinning counter-clockwise. 
+Now that the module is armed, any new throttle commands sent using the *ESC Input* parameter will change how it is spinning.
+For example, setting *ESC Input* to 0.1 should set the module to spin with a drive voltage of 1V. This is because the module is in Voltage mode with a *Max Volts* of 10V, so a 
+10% throttle command commands 1V. Confirm that the module is spinning counter-clockwise, matching the direction that was configured previously.
 
-When you are done with the testing, set the Coast parameter to stop the motor. Now, you can move on to setting up the flight controller.
+For more information on how to configure the module to properly interpret throttle commands, see the :ref:`manual_throttle` section of the Feature Reference 
+Manual. For more information on arming and disarming the module, refer to the :ref:`manual_advanced_arming` section of the Feature Reference Manual.
+
+If the module is spinning as expected, set the *Coast* parameter to stop the module. Now, you can move on to setting up the flight controller.
```

Comment:
With your module configured to spin, you must now set up your flight controller to send proper commands.